lib/trino-filesystem-manager/src/main/java/io/trino/filesystem/manager/FileSystemModule.java

```diff
@@ -127,6 +137,9 @@ public TrinoFileSystemFactory createFileSystemFactory(
         if (fileSystemCache.isPresent()) {
             delegate = new CacheFileSystemFactory(tracer, delegate, fileSystemCache.orElseThrow(), keyProvider.orElseThrow());
         }
+        if (memoryFileSystemCache.isPresent()) {
```

Comment:
I would say if `iceberg.metadata-cache.enabled=true` but alluxio is already there, you don't have to wrap with mem. This is because Alluxio already satisfies `iceberg.metadata-cache.enabled` requirement.

Reply:
removed the wrapping, now the in-memory cache will be ignored when alluxio is enabled